### PR TITLE
docs(resource-detector-alibaba): add Semantic Conventions to alibaba README

### PR DESCRIPTION
**Pull request by maryliag**
## Which problem is this PR solving?

- Add Semantic Conventions to Alibaba README

## Short description of the changes

- Part Of #2025
- Add columns of semantic conventions to README

### CODE REVIEW

File: detectors/node/opentelemetry-resource-detector-alibaba-cloud/README.md [hunk 1]
```diff
@@ -26,21 +26,23 @@ const resource = await detectResources({
...
+| cloud.account.id        | Value of `owner-account-id` on Alibaba Cloud                    | Key: `SEMRESATTRS_CLOUD_ACCOUNT_ID`        |
+| cloud.availability_zone | Value of `zone-id` on Alibaba Cloud                             | Key: `SEMRESATTRS_CLOUD_AVAILABILITY_ZONE` |
+| cloud.platform          | In this context, it's always `alibaba_cloud_ecs`                | Key: `SEMRESATTRS_CLOUD_PLATFORM`          |
+| cloud.provider          | In this context, it's always `alibaba_cloud`                    | Key: `SEMRESATTRS_CLOUD_PROVIDER`          |
+| cloud.region            | Value of `region-id` on Alibaba Cloud                           | Key: `SEMRESATTRS_CLOUD_REGION`            |
+| host.id                 | Value of `instance-id` on Alibaba Cloud                         | Key: `SEMRESATTRS_HOST_ID`                 |
+| host.name               | The hostname for the app, retrieve from the `hostname` endpoint | Key: `SEMRESATTRS_HOST_TYPE`               |
+| host.type               | Value of `instance-type` on Alibaba Cloud                       | Key: `SEMRESATTRS_HOST_NAME`               |
```
Comment (JamieDanielson) [inline detectors/node/opentelemetry-resource-detector-alibaba-cloud/README.md:46]: Based on comments over on #1778 we may decide not to move forward with this Notes section as it is extra to maintain.

**Comment by maryliag** [comment]
sounds good, will close this

**Comment by JamieDanielson** [comment]
Oh! I should note, I think your other section in here referencing the specific semantic convention version is okay to keep - just the Keys in the table are not needed. Want to re-open this or open a new PR for that @maryliag ? The original updates were on #2046 but that was before we started combining the two updates together, so it looks like we still want to mention the version in use in the README.

**Comment by maryliag** [comment]
Ah okay, thanks @JamieDanielson , I created a new PR for it: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2130

**Pull request closed without merging**